src/Console/FetchDataFromOzu.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace Code16\OzuClient\Console;
+
+use Code16\OzuClient\Client;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Artisan;
+use Symfony\Component\Process\Process;
+use ZipArchive;
+
+class FetchDataFromOzu extends Command
+{
+    protected $signature = 'ozu:fetch-ozu-data';
+
+    protected $description = 'Gets Ozu’s CMS data and replaces your local database and assets with it.';
+
+    private string $databaseDumpPath;
+    private string $assetsZipPath;
+    private string $assetsExtractPath;
+
+    public function handle(Client $ozuClient): int
+    {
+
+        $this->warn('/!\\ This action will erase your local database and assets.');
+
+        if (!$this->confirm("Are you sure you want to continue? This cannot be undone.")) {
+            return self::SUCCESS;
+        }
+
+        $this->initializePaths();
+
+        if (!$this->downloadDatabase($ozuClient)) {
+            return self::FAILURE;
+        }
+
+        if (!$this->importDatabase()) {
+            return self::FAILURE;
+        }
+
+        if (!$this->downloadAssets($ozuClient)) {
+            return self::FAILURE;
+        }
+
+        if (!$this->extractAssets()) {
+            return self::FAILURE;
+        }
+
+        $this->cleanTemporaryFiles();
+        $this->info('✅ Ozu data successfully imported.');
+        return self::SUCCESS;
+    }
+
+    private function initializePaths(): void
+    {
+        $this->databaseDumpPath = storage_path('app/tmp/ozu.sql');
+        $this->assetsZipPath = storage_path('app/tmp/ozu-assets.zip');
+        $this->assetsExtractPath = storage_path('app/public/data/' . config('ozu-client.website_key') . '/');
+    }
+
+    private function downloadDatabase(Client $ozuClient): bool
+    {
+        $this->info('⬇️ Downloading Ozu database...');
+        $dbPath = $ozuClient->downloadOzuDatabase();
+
+        if (!$dbPath) {
+            $this->error('❌ Failed to download the Ozu database.');
+            return false;
+        }
+
+        $this->databaseDumpPath = $dbPath;
+        return true;
+    }
+
+    private function importDatabase(): bool
+    {
+        $this->info('🧹 Wiping local database...');
+        Artisan::call('db:wipe');
+
+        $this->info('🛠 Importing Ozu database...');
+        $config = config('database.connections.mysql');
+
+        $process = Process::fromShellCommandline(sprintf(
```

Comment:
Instead of running a mysql command use [`DB::unprepared()` ](https://laravel.com/docs/12.x/database#running-an-unprepared-statement) method